### PR TITLE
Add team option for SSH keys

### PR DESCRIPTION
**Pull request by contolini**
@vickumar1981 @marcesher @imuchnik  Couple things happening here:

1. Moved the public key storage functionality to a [standalone script](https://github.com/catops/catops-keys) so that it can be used with other tools.
1. Added a `team` option. If specified, the public keys of all users on that team will be added to the server. Teams are managed with [catops-teams](https://github.com/catops/catops-teams).
1. Added a `HUBOT_AWS_EC2_REQUIRE_SSH_KEY` to make the key requirement optional. This is set to `true` in our environment file.

### CODE REVIEW

**Comment by vickumar1981** [comment]
👍

File: scripts/ec2/run.coffee [hunk 1]
```diff
@@ -75,6 +65,22 @@ module.exports = (robot) ->
...
+        if member and member.key
+          ssh_key += member.key + '\n'
+        return
+    # If we're not using teams, check for the user's key
+    else if robot.brain.userForId(msg.envelope.user.id).key
+      ssh_key = robot.brain.userForId(msg.envelope.user.id).key
+
+    # If a key is required and we didn't find one, abort
+    if !!process.env.HUBOT_AWS_EC2_REQUIRE_SSH_KEY and !ssh_key.length
+      return msg.send "You need to set your SSH *public* key first. To do so, copy your ~/.ssh/id_rsa.pub into your clipboard, and then in chat run `#{robot.name} my key is [your_ssh_key]`"
```
Comment (marcesher) [inline scripts/ec2/run.coffee]: @contolini shouldn't this message be `my public key is...` ?

Comment (contolini) [inline scripts/ec2/run.coffee]: Yuuuup.

**Comment by imuchnik** [comment]
All the cool stuff happening when I am on vacation? I sure hope the governance/maturity stuff is done ;)

**Pull request merged**